design/design-foundry_image_guidelines.md

```diff
@@ -0,0 +1,191 @@
+
+
+### 4. **Use a 10px scale when sizing shapes and strokes.**
+
+For example, if you are using two circles, a large and small, try sizing them at 40px and 10px. Use the same scale for applying strokes to objects (always set the stroke on the inner or outer edge) with the default stroke weight being 20px. This maintains consistency and helps us create images quickly.
+
+
+### No-no’s:
+- Don’t use text in general post images
+- Don’t use texture
+- Avoid blending modes like multiply or overlay—they tend to skew the color palette (transparency is okay)
+- Don’t create complex backgrounds or environments
+
+
+### Feel Free to:
+- Use transparency to create variation in the color palette
+- Use a pattern
+- Create recognizable objects or use icons that fit in the simplified style
+
+When starting a post image, use this starter Illustrator file.
+https://www.dropbox.com/s/rx1m22xgybmh2mk/-Foundry%20Image%20Starter.ai?dl=0
+
+The file above contains pre-created layers with each category background color. Designers can turn off the ones they don’t need, and they’re ready to go. Some layers also contain image examples, the color guide, and pre-populated color swatches.
+
+![](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560350441871_Screen+Shot+2019-06-12+at+10.40.26+AM.png)
+
+### Examples
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1548349316084_Screen+Shot+2019-01-24+at+12.01.37+PM.png)
+
+
+
+## Case Study Images
+
+A case study image will belong in one of the four Foundry categories: UX & Content, Design, Development, or Culture. This is because a case study typically focuses on one of these areas. Identify the appropriate case study layer in the starter file that contains the gradient stroke for that category. Place the client’s logo centered in the frame. Be sure to use ample white space around the logo.
+
+
+![This is a case study that belongs in the  Design category.](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560351236090_case-study_boom-crate.jpg)
+This is a case study that belongs in the Design category.
+
+![This is a case study that belongs in  the Development category.](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560351250646_case-study_dealer-tire-01.jpg)
+This is a case study that belongs in the Development category.
+
+
+
+## Meet the Team Images
+
+Meet the team images are used with Foundry posts that introduce new team members. They exist in the Culture category, which uses blue from the color palette. When creating these images, open the starter file…
+
+https://www.dropbox.com/s/aywolmsfn1pmrm4/Foundry%20Team%20Image.psd?dl=0
```

Comment:
A lot of the images in here are showing up in various sizes, some very large. Do we need to reduce the sizes for this Standard file?

Reply:
These are the actual sizes they're exported at, but rarely show this large on the foundry. I'm ok with leaving them since formatting options in GitHub leave much to be desired anyway.